tools/solidityUpgrade/SourceTransform.h

```diff
@@ -82,6 +104,7 @@ class SourceGeneration
 		for (auto inheritedContract: _contracts)
 			overrideList += inheritedContract->name() + ",";
 
+		// Note: Can create incorrect replacements
```

Comment:
Can you create a separate issue for that and a case where it happens?

Reply:
I ran into this when running on external-tests (colonyNetwork.) I'll run it again, and make an issue with minimal example.

Reply:
Here is an example: https://github.com/ethereum/solidity/issues/8786